### PR TITLE
Feature/fix codecoverage for npm proxy

### PR DESCRIPTION
**Pull request by brastak**
For #30

### CODE REVIEW

**Comment by 0crat** [comment]
This pull request [#31](https://github.com/artipie/npm-proxy-adapter/issues/31) is assigned to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/5267e3a9-193c-46ef-a483-20114c80996a); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: src/test/java/com/artipie/npm/proxy/json/CachedContentTest.java [hunk 1]
```diff
@@ -47,13 +46,7 @@ public void getsValue() throws IOException {
...
         final DocumentContext json = JsonPath.parse(transformed);
-        final JSONArray refs = json.read("$.versions.[*].dist.tarball", JSONArray.class);
-        MatcherAssert.assertThat("Could not find asset references", refs.size() > 0);
-        for (final Object ref: refs) {
-            MatcherAssert.assertThat(
-                (String) ref,
-                new StringStartsWith(TransformedContent.PLACEHOLDER)
-            );
-        }
+        final String ref = json.read("$.versions.['1.0.0'].dist.tarball", String.class);
```
Comment (carlosmiranda) [inline src/test/java/com/artipie/npm/proxy/json/CachedContentTest.java]: @brastak let's inline this variable.

**Review by carlosmiranda — CHANGES_REQUESTED** [review]
@brastak just one comment

File: src/test/java/com/artipie/npm/proxy/json/CachedContentTest.java (hunk 1 above)
Comment (brastak) [reply src/test/java/com/artipie/npm/proxy/json/CachedContentTest.java]: @carlosmiranda done

**Comment by brastak** [comment]
@carlosmiranda please, see update

**Comment by brastak** [comment]
@carlosmiranda ping

**Review by carlosmiranda — APPROVED** [review]
@brastak thanks. @g4s8 this looks good from my end

**Review by g4s8 — APPROVED** [review]

**Comment by 0crat** [comment]
Job `gh:artipie/npm-proxy-adapter#31` is not assigned, can't get performer

**Pull request merged**